frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +258,26 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    local info = Notification:new{text = _("Changing font size")}
```

Comment:
> Could you add a period please? :-)

I think it isn't necessary here. "Changing font size" is shown until font is changed and next notification "Font sieze set to %1" show.
> So first it shows "Changing font size" and shortly thereafter "Font size set to %1."?

Yes first it shows "Changing font size" and next "Font size set to..."